src/main/java/pl/edu/zut/mad/schedule/util/Extensions.kt

```diff
@@ -2,10 +2,15 @@ package pl.edu.zut.mad.schedule.util
 
 import android.app.Activity
 import android.support.v4.app.Fragment
+import android.util.Log
 import pl.edu.zut.mad.schedule.app.ScheduleApp
 
 val Activity.app: ScheduleApp
     get() = application as ScheduleApp
 
 val Fragment.app: ScheduleApp
     get() = activity.app
+
+fun Any.log(text: String) {
```

Comment:
Be aware that this is just static method. When used in class it would require to use Robolectric (or PowerMockito even worse) to test it because of "Log" usage

Reply:
This is only for debug purposes, I will not pushing any code to master branch with using this extension.